### DISCUSSION TITLE
Problem in threading firmware of pyboard v1.1

### DISCUSSION THREAD

**Discussion by IlysvlVEizbr** [micropython/micropython#9093] [Using MicroPython] [answered]
User can't even boot the board (sometimes after disconnect the power for a while and then reconnect it). I think this could be considered as a serious problem, the bug exist from v1.17 release and it's still there now v1.19.1. I try to clone the repository and build it in Ubuntu 20.04, to see what's going on, but fail to build it with the steps in readme (even without any modify to the code).

Can any developer see this, and check what would be the problem?

For detail, see #8528.

As I saw a discussion about moving the forum to here, so I move the unresolved [topic](https://forum.micropython.org/viewtopic.php?f=6&t=12484) to here.

**Comment by jimmo** [comment, marked as answer]
Thanks @IlysvlVEizbr we're experimenting with setting up Discussions, but haven't moved over yet.

I was able to reproduce this. I will update #7816 with more information.

**Reply by jimmo** [reply]
Should be fixed by https://github.com/micropython/micropython/pull/9094

**Reply by IlysvlVEizbr** [reply]
> Should be fixed by #9094

The new bug is: `_thread` module is lost.
There is no `_thread ` module in firmware downloaded from [MicroPython homepage](https://micropython.org/resources/firmware/pybv11-thread-20220906-unstable-v1.19.1-375-ge90b85cc9.dfu)
```py
>>> import _thread
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: no module named '_thread'
```

**Reply by jimmo** [reply]
Argh, yes I see the problem. PR incoming.

**Reply by jimmo** [reply]
Thanks for reporting. Fixed in https://github.com/micropython/micropython/pull/9247
